analyses/chromosomal-instability/util/chr-break-calculate.R

```diff
@@ -0,0 +1,253 @@
+# Functions for chromosomal instability plots
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2020
+
+make_granges <- function(break_df = NULL,
+                         sample_id = NULL,
+                         samples_col = "samples",
+                         chrom_col = "chrom",
+                         start_col = "start",
+                         end_col = "end") {
+  # For a given breaks data.frame make a GenomicRanger object from it. Can
+  # filter to a single samples' data.
+  #
+  # Args:
+  #   break_df: for a data.frame with chromosomal coordinates and sample IDs.
+  #   sample_id: a character string that designates which data needs to be
+  #              extracted and made intoa GenomicRanges object by matching the
+  #              information in the designated sample_col. If "all" is designated,
+  #              all samples will be kept. Multiple samples can be designated
+  #              as a character vector.
+  #   samples_col: character string that indicates the column name with the
+  #                sample ID information. Default is "samples". This information
+  #                will be store in the GenomicRanges object in
+  #                `elementData@listData$samples`
+  #   chrom_col: character string that indicates the column name with the
+  #              chromosome information. Default is "chrom".
+  #   start_col: character string that indicates the column name with the
+  #              start coordinate. Default is "start".
+  #   end_col: character string that indicates the column name with the
+  #            end coordinate. Default is "end".
+  #
+  # Returns:
+  # A Genomic Ranges formatted object for the particular sample that has the
```

Comment:
Is this more appropriate?
```suggestion
  # A GRanges object for the particular sample that has the
```